### PR TITLE
added application

### PR DESCRIPTION
**Pull request by hoseuz**

### CODE REVIEW

File: person.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+ class Person
+{
+    public string FirstName { get; set; }
+    public string LastName { get; set; }
+    public DateTime BirthDate { get; set; }
+    public string Address { get; set; }
+
```
Comment (IvanParaskevov) [inline person.cs:7]: Very cool

**Review by IvanParaskevov — APPROVED** [review]
Very good

File: application.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class EducationalProgram
+{
+    static void Main()
+    {
+        StudentAndTeacherEntity studentAndTeacher1 = new StudentAndTeacherEntity("John", "Doe", new DateTime(1995, 5, 15), "123 Main St", 12345, "Computer Science", 3.75, 2, "Science");
+        StudentAndTeacherEntity studentAndTeacher2 = new StudentAndTeacherEntity("Alice", "Johnson", new DateTime(1998, 10, 8), "456 Elm St", 54321, "Mathematics", 3.90, 5, "Mathematics");
+        AttendanceStatistics statistics = new AttendanceStatistics();
+
+        studentAndTeacher1.AttendClass();
```
Comment (dchoynev) [inline application.cs:9]: congrats

**Review by dchoynev — COMMENTED** [review]
1212

**Review by dchoynev — APPROVED** [review]

**Pull request merged**